src/lib.rs

```diff
@@ -10,18 +10,17 @@ use unicode_width::UnicodeWidthStr;
 // Constants! :D
 const ENDSL: &[u8] = b"| ";
 const ENDSR: &[u8] = b" |\n";
-#[cfg(not(feature = "clippy"))]
-const FERRIS: &[u8] = br#"
+const MASCOT: &[u8] = if !cfg!(feature = "clippy") {
```

Comment:
consider flipping the branches instead of inverting the condition

Reply:
Done. Good call.